### DISCUSSION TITLE
Priority List for people wanting to help

### DISCUSSION THREAD

**Discussion by derrohrbach** [FossifyOrg/General-Discussion#91] [General]
Do you have a priority list somewhere of issues/features that devs like me can and should work on first? Would be very handy, since I'd like to help out with future development of this project

**Comment by inson1** [comment]
likes and stars are good indicators, somethings with good value per hour, also depends on how much time and experience you have
non device specific reproducible bugs should have priority

just work on something that you would like to see and use

**Comment by Aga-C** [comment]
Currently, there's no priority list, but you can look at it this way:
- Issues labeled as ASAP are the top priority.
- Then there are all other issues, from which do whatever is most important for you. I wouldn't prioritize there any bugs or feature request - any help is welcome. Just please don't take issues labeled as `needs triage`.

**Reply by gmasil** [reply]
I think we should have a label that indicates a feature request is approved by the maintainers. Currently I wouldn't just want to start working on a feature request without knowing if that feature is desired by the team in general. On SMT I was occasionally asking if the feature is desired, without ever getting an answer. This really drags down the motivation to help out tbh.

**Reply by inson1** [reply]
@gmasil I think all open and without triage issues are approved, at least that is my understanding :)
